src/core/connection.h

```diff
@@ -630,6 +630,11 @@ typedef struct QUIC_CONNECTION {
     //
     uint16_t KeepAlivePadding;
 
+    //
+    // Indicates that the peer accepts RELIABLE_RESET kind of frames, in addition to RESET_STREAM frames.
+    //
+    uint8_t ReliableResetStreamNegotiated;
```

Comment:
Let's put this in `QUIC_CONNECTION_STATE` instead of here.

Reply:
Added, will update, and I can change its type to BOOLEAN.